lib/kube/proxy/utils_test.go

```diff
@@ -293,7 +293,7 @@ func (c *testContext) genTestKubeClientTLSCert(t *testing.T, userName, kubeClust
 	}, true)
 	require.NoError(t, err)
 
-	caCert, signer, err := authServer.GetKeyStore().GetTLSCertAndSigner(ca)
+	caCert, signer, err := authServer.GetKeyStore().GetTLSCertAndSigner(context.Background(), ca)
```

Comment:
Can you create a context at the start of this method, and then use it for the `GetCertAuthority` call on line 290 as well ?